plugins/backend/static/js/logics_blockly_code.js

```diff
@@ -0,0 +1,277 @@
+/**

```

Comment:
The JS code to run Blockly inside the html page. Mostly copy&paste; needs refactoring to better fit into here.